src/control/system/member.go

```diff
@@ -35,50 +35,69 @@ import (
 )
 
 // MemberState represents the activity state of DAOS system members.
-//go:generate stringer -type=MemberState
 type MemberState int
 
 const (
-	Started MemberState = iota
-	Stopping
-	Errored
-	Unresponsive
+	MemberStateUnknown MemberState = iota
+	MemberStateStarted
+	MemberStateStopping
+	MemberStateErrored
+	MemberStateUnresponsive
 )
 
-// SystemMember refers to a data-plane instance that is a member of this DAOS
+func (ms MemberState) String() string {
+	return [...]string{"Unknown", "Started", "Stopping", "Errored", "Unresponsive"}[ms]
+}
+
+// Member refers to a data-plane instance that is a member of this DAOS
 // system running on host with the control-plane listening at "Addr".
-type SystemMember struct {
+type Member struct {
 	Rank  uint32
 	UUID  string
 	Addr  net.Addr
-	State MemberState
+	state MemberState
```

Comment:
Is there a plan to protect this field with a mutex?

Reply:
yes although it currently is only ever accessed through membership which is protected by it's own mutex. do you not think that is sufficient?

Reply:
Hmm, probably fine for now.